src/System.Drawing.Common/src/System/Drawing/Printing/PaperSourceKind.cs

```diff
@@ -0,0 +1,132 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Drawing.Printing
+{
+    /// <include file='doc\PaperSourceKind.uex' path='docs/doc[@for="PaperSourceKind"]/*' />
+    /// <devdoc>
+    ///    <para>
+    ///       Standard paper sources.
+    ///    </para>
+    /// </devdoc>
+    [Serializable]
+    public enum PaperSourceKind
+    {
+        // Please keep these in SafeNativeMethods.cs order
+
+        /// <include file='doc\PaperSourceKind.uex' path='docs/doc[@for="PaperSourceKind.Upper"]/*' />
+        /// <devdoc>
+        ///    <para>
+        ///       The upper bin of a printer (or, if the printer only has one bin, the only bin).
+        ///    </para>
+        /// </devdoc>
+        Upper = SafeNativeMethods.DMBIN_UPPER,
+
+        /// <include file='doc\PaperSourceKind.uex' path='docs/doc[@for="PaperSourceKind.Lower"]/*' />
+        /// <devdoc>
+        ///    <para>
+        ///       The lower bin of a printer.
+        ///    </para>
+        /// </devdoc>
+        Lower = SafeNativeMethods.DMBIN_LOWER,
+
+        /// <include file='doc\PaperSourceKind.uex' path='docs/doc[@for="PaperSourceKind.Middle"]/*' />
+        /// <devdoc>
+        ///    <para>
+        ///       The middle bin of a printer.
+        ///    </para>
+        /// </devdoc>
+        Middle = SafeNativeMethods.DMBIN_MIDDLE,
+
+        /// <include file='doc\PaperSourceKind.uex' path='docs/doc[@for="PaperSourceKind.Manual"]/*' />
+        /// <devdoc>
+        ///    <para>
+        ///       Manually-fed paper.
+        ///    </para>
+        /// </devdoc>
+        Manual = SafeNativeMethods.DMBIN_MANUAL,
+
+        /// <include file='doc\PaperSourceKind.uex' path='docs/doc[@for="PaperSourceKind.Envelope"]/*' />
```

Comment:
We don't. Cleaning these up is on my list of follow-up items, which I'll make sure to file tomorrow. Unfortunately, this style of comment is pretty common in our .NET Framework codebase; I'm not even sure what the names and paths refer to.